akka-http/src/main/scala/akka/http/scaladsl/unmarshalling/PredefinedFromStringUnmarshallers.scala

```diff
@@ -78,11 +66,39 @@ trait PredefinedFromStringUnmarshallers {
       }
     }
 
-  val CsvString: Unmarshaller[String, immutable.Seq[String]] =
+  val CsvStringSeq: Unmarshaller[String, immutable.Seq[String]] =
     Unmarshaller.strict[String, immutable.Seq[String]] { string ⇒
       string.split(",").toList
     }
 
+  val CsvByteSeq: Unmarshaller[String, immutable.Seq[Byte]] =
+    CsvStringSeq.map(_.map(byteFromString))
+
+  val CsvShortSeq: Unmarshaller[String, immutable.Seq[Short]] =
+    CsvStringSeq.map(_.map(shortFromString))
+
+  val CsvIntSeq: Unmarshaller[String, immutable.Seq[Int]] =
+    CsvStringSeq.map(_.map(intFromString))
+
+  val CsvLongSeq: Unmarshaller[String, immutable.Seq[Long]] =
+    CsvStringSeq.map(_.map(longFromString))
```

Comment:
+1, this is not meant to "solve all the things". This is meant to solve the typical: `ids=1,2,3,4` or `names=bob,me,myself`, the numeric one allowing to be Int/Long/... etc